dropwizard-parent-pom/pom.xml

```diff
@@ -252,11 +251,6 @@
       </dependency>
       <dependency>
         <groupId>io.dropwizard</groupId>
-        <artifactId>dropwizard-spdy</artifactId>
```

Comment:
It doesn't look like you went through the dropwizard parent pom and pinned every artifact in there to the same version.  You should look [here](https://repo1.maven.org/maven2/io/dropwizard/dropwizard-parent/1.0.2/dropwizard-parent-1.0.2.pom) and make sure you mention every artifact that Dropwizard defines.  Some old ones were removed in 1.0.x and some new ones were added.